src/FSAL/FSAL_SCALITY/dbd_rest_client.c

```diff
@@ -547,6 +550,44 @@ dbd_readdir(struct scality_fsal_export* export,
 	return;
 }
 
+int
+dbd_collect_bucket_attributes(struct scality_fsal_export *export)
+{
+        int ret = 0;
+        dbd_response_t *response;
+        response = dbd_get(export, ATTRIBUTES_BASE_PATH, "", NULL);
+        if ( NULL != response && 200 == response->http_status ) {
+		json_t *js_owner = json_object_get(response->body, "owner");
+		json_t *js_owner_display_name = json_object_get(response->body, "ownerDisplayName");
+                const char *owner = NULL,
+                        *owner_display_name = NULL;
+                if (js_owner)
+                        owner = json_string_value(js_owner);
+                if (js_owner_display_name)
+                        owner_display_name = json_string_value(js_owner_display_name);
```

Comment:
At this stage of development, the ACLs are not yet integrated to NFS. Object created through NFS are assigned to the bucket owner and no authentication is done